src/vs/editor/contrib/find/browser/findController.ts

```diff
@@ -739,19 +739,15 @@ export abstract class SelectionMatchFindAction extends EditorAction {
 			return;
 		}
 
-		const seedSearchStringFromNonEmptySelection = editor.getOption(EditorOption.find).seedSearchStringFromSelection === 'selection';
-		let selectionSearchString = null;
-		if (editor.getOption(EditorOption.find).seedSearchStringFromSelection !== 'never') {
-			selectionSearchString = getSelectionSearchString(editor, 'single', seedSearchStringFromNonEmptySelection);
-		}
+		const selectionSearchString = getSelectionSearchString(editor, 'single', false);
```

Comment:
seeding the search from cursor position alone should still be possible if `seedSearchStringFromSelection` is set to always.  Hard coding the last parameter to false will break that scenario

Reply:
note that I meant the new code to behave _exactly_ as the old code with `seedSearchStringFromSelection==='always'`; now it passes `'none'` instead of `'single'` at line 749, but that was effectively a no-op as far as I can tell

Reply:
from looking at `getSelectionSearchString`, this change will require that text is selected for it to be seeded as the search string, whereas currently the word at the cursor location will be used as the seed if nothing is selected

Reply:
okay, so I compiled this branch and here is a screencast: Ctrl+F3 definitely picks up the word at cursor without requiring a selection...  Note that this behaviour is different from the current version mainline, where "inline" will be searched instead of "never" upon Ctrl+F3 with these settings.
![seed](https://user-images.githubusercontent.com/53914057/207990344-47d68d77-2119-4cea-a737-06c198832622.gif)

Reply:
Here is a separate issue that I see though, where the Find Widget is not populated — but the search is still seeded!  Just to make sure to not mix it up with my issue...
It happens (1) only on the very first invocation after startup (2) if the find widget is closed, and (3) the same happens on the mainline version with the default settings — so should be out of scope for this issue.
![seed-atFirst](https://user-images.githubusercontent.com/53914057/207994201-e2213cf6-0f53-4833-b671-59f2efb07099.gif)

Reply:
ok, you were right, I still had it backwards. This will _always_ use the current word as the seed when nothing is selected, regardless of the settings. Seems ok since the regular find command is configurable to only seed when something is selected.